ui/src/main/webapp/src/app/app.routing.ts

```diff
@@ -98,7 +98,7 @@ export const appRoutes: Routes = [
                             wizard: true
                         },
                         children: [
-                            { path: 'create-project', component: MigrationProjectFormComponent, data: {displayName: 'Create new Project', wizard: true} },
+                            { path: 'create-project', component: MigrationProjectFormComponent, data: {displayName: 'Create new Project', wizard: true, creating: true} },
```

Comment:
I haven't read how this is used yet, but this seems wrong. We already have an editMode flag in the component.